### PR TITLE
remove duplicate code

### PR DESCRIPTION
**Pull request by FabianTerhorst**

### CODE REVIEW

File: library/src/main/java/com/pokegoapi/api/PokemonGo.java [hunk 1]
```diff
@@ -223,17 +223,8 @@ private void fireRequestBlockOne() throws RemoteServerException, LoginFailedExce
...
-
-		getRequestHandler().sendServerRequests(requests);
-		try {
-			inventories.updateInventories(GetInventoryResponse.parseFrom(requests[2].getData()));
-			settings.updateSettings(DownloadSettingsResponse.parseFrom(requests[4].getData()));
-		} catch (InvalidProtocolBufferException e) {
-			throw new RemoteServerException();
-		}
+		fireRequestBlock(new ServerRequest(RequestTypeOuterClass.RequestType.GET_ASSET_DIGEST,
+				CommonRequest.getGetAssetDigestMessageRequest()));
```
Comment (iGio90) [inline library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: we will need to add here the code from @ramarro123 to parse the response and update the classes

Comment (FabianTerhorst) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: and where is the problem?

Comment (iGio90) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: nowhere :D merge it, gonna add it once his pr is ready

Comment (iGio90) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: aw wait!

Comment (iGio90) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: how we will access to the responses?

Comment (iGio90) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: you need to define a variable so we can access it later to parse the responses

Comment (FabianTerhorst) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: just edit it when its merged and we realy need a variable

Comment (FabianTerhorst) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: in the current code we dont need this

Comment (iGio90) [reply library/src/main/java/com/pokegoapi/api/PokemonGo.java:227]: ok, i'll do it :P merge this

**Pull request merged**